### PR TITLE
refactor: add code-pushup.preset.ts

### PR DESCRIPTION
**Pull request by BioPhoton**
This PR adds a `code-pushup.preset.ts` and uses the preset in `code-pushup.config.ts`

Notice: 
Coverage helper still unfinished.

### CODE REVIEW

File: code-pushup.preset.ts [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+): Promise<CoreConfig> => {
+  const targetNames = ['unit-test', 'integration-test'];
+  const targetArgs = [
+    '-t',
+    'unit-test',
+    'integration-test',
+    '--coverage.enabled',
+    '--skipNxCache',
+  ];
+  const rootArgs = ['nx', 'run-many'];
```
Comment (matejchalk) [inline code-pushup.preset.ts:145]: Unused variable:

```suggestion
```

File: code-pushup.config.ts [hunk 2]
```diff
@@ -72,88 +66,10 @@ const config: CoreConfig = {
...
-      refs: [
-        {
-          type: 'group',
-          plugin: 'js-packages',
-          slug: 'npm-outdated',
-          weight: 1,
-        },
-      ],
-    },
+    ...coverageCategories,
```
Comment (matejchalk) [inline code-pushup.config.ts:72]: Why not add `coverageCoreConfigNx()` to `mergeConfigs` instead? Otherwise I don't know why you created the function.

Comment (BioPhoton) [reply code-pushup.config.ts:72]: I have to remove the fn I guess or implement a version that also works for project level. I did not find a helper for that in the coverage plugin.

File: code-pushup.preset.ts (hunk 1 above)
Comment (BioPhoton) [reply code-pushup.preset.ts:145]: removed

File: code-pushup.config.ts (hunk 2 above)
Comment (BioPhoton) [reply code-pushup.config.ts:72]: "implemented" :)

**Comment by BioPhoton** [comment]
Sry for the mess in the first version of the PR!

File: code-pushup.preset.ts (hunk 1 above)
Comment (matejchalk) [reply code-pushup.preset.ts:145]: You haven't pushed anything since my last review :sweat_smile:

**Review by matejchalk — APPROVED** [review]

File: packages/utils/src/lib/merge-configs.unit.test.ts [hunk 3]
```diff
@@ -172,7 +172,7 @@ describe('mergeObjects', () => {
     });
   });
 
-  it('should merge add upload to config', () => {
+  it('should merge upload config properties', () => {
```
Comment (vmasek) [inline packages/utils/src/lib/merge-configs.unit.test.ts:175]: 👍

**Review by vmasek — APPROVED** [review]

**Pull request merged**